lib/pages/detail_movie_page.dart

```diff
@@ -1,33 +1,45 @@
 import 'package:flutter/material.dart';
 
-import 'search_engine.dart';
-import 'styles.dart';
-import 'button_section.dart';
-import 'movie.dart';
-import 'movie_details.dart';
-import 'movie_genres.dart';
-import 'movie_header.dart';
-import 'movie_presentation.dart';
+import '../Widgets/search_engine.dart';
+import '../utils/styles.dart';
+import '../Widgets/button_section.dart';
+import '../models/movie.dart';
+import '../Widgets/movie_details.dart';
+import '../Widgets/movie_header.dart';
+import '../Widgets/movie_presentation.dart';
+import '../widgets/genres_details_page.dart';
 
-class MyHomePage extends StatefulWidget {
-  const MyHomePage({super.key, required this.title});
+class DetailMoviePage extends StatefulWidget {
```

Comment:
Why you need this widget to be `Stateful`? I don't see any inner state!

Reply:
For the button section on the movie details page

Reply:
That's right! I used it before but now it doesn't have any sense! Done